src/components/EachContinent.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { FaArrowLeft, FaSearch } from 'react-icons/fa';
+import NavBar from './shared/NavBar';
+import { fetchRegionDetails } from '../redux/countries';
+
+const EachContent = () => {
+  let regionDetails = useSelector((state) => state.entities.countries);
+
+  const { title } = useParams();
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchRegionDetails(title));
+  }, []);
+
+  const [searchText, setYourFilterText] = useState('');
```

Comment:
- Well done so far implementing the changes requested by the previous reviewer including the README file and the search functionality. However, the search isn't functioning in the live demo link. Please fix this problem to get the approval 🙏🏻